vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.ts

```diff
@@ -14,45 +14,8 @@ window.Vaadin.Flow.gridConnector.initLazy = (grid) => {
 
   const dataProviderController = grid._dataProviderController;
 
-  dataProviderController.ensureFlatIndexHierarchyOriginal = dataProviderController.ensureFlatIndexHierarchy;
-  dataProviderController.ensureFlatIndexHierarchy = function (flatIndex) {
-    const { item } = this.getFlatIndexContext(flatIndex);
-    if (!item || !this.isExpanded(item)) {
-      return;
-    }
-
-    const isCached = grid.$connector.hasCacheForParentKey(grid.getItemId(item));
-    if (isCached) {
-      // The sub-cache items are already in the connector's cache. Skip the debouncing process.
-      this.ensureFlatIndexHierarchyOriginal(flatIndex);
-    } else {
-      grid.$connector.beforeEnsureFlatIndexHierarchy(flatIndex, item);
-    }
-  };
-
-  dataProviderController.isLoadingOriginal = dataProviderController.isLoading;
-  dataProviderController.isLoading = function () {
-    return grid.$connector.hasEnsureSubCacheQueue() || this.isLoadingOriginal();
-  };
-
-  dataProviderController.getItemSubCache = function (item) {
-    return this.getItemContext(item)?.subCache;
-  };
-
   let cache = {};
```

Comment:
Technically, this internal cache is now redundant, and gridConnector could be refactored to add items directly to the web component's cache (dataProviderController). However, I'd prefer to improve that separately when this refactoring is completed.

Reply:
Is there a separate PR or issue to address this? Just want to make sure we don't forget this part.